android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -142,6 +142,8 @@ internal fun mapFromDeviceType(type: DeviceType): String {
         DeviceType.STRIPE_M2 -> "stripeM2"
         DeviceType.STRIPE_S700 -> "stripeS700"
         DeviceType.STRIPE_S700_DEVKIT -> "stripeS700Devkit"
+        DeviceType.STRIPE_S710 -> "stripeS710"
```

Comment:
are these not needed for iOS?

Reply:
yes but iOS does't support S710 yet by v3.8.0.